### DISCUSSION TITLE
Import typescript yarn monorepo module

### DISCUSSION THREAD

**Discussion by Hurtak** [vercel/next.js#13500] [Help]
I have yarn monorepo with following structure

```
packages/
  nextapp/
  shared/
```

Where from nextapp I do 

```
import x from "@monorepo/shared"
```

It resolves the package correctly, but since these are TS files, the TS loader is not applied and it fails with `Module parse failed: Unexpected token`. Any idea how to modify webpack config to make this work?

**Comment by matamatanot** [comment]
FYR
https://github.com/vercel/next.js/pull/13542/files#diff-f984cc361ad44d86838abd691424d44d
